sktime/transformations/panel/tsfresh.py

```diff
@@ -33,7 +33,7 @@ class _TSFreshFeatureExtractor(BaseTransformer):
 
     def __init__(
         self,
-        default_fc_parameters="efficient",
+        default_fc_parameters=None,
```

Comment:
If `None` has a different output from `"efficient"` this will require deprecation.